.github/workflows/build-base.yaml

```
@@ -10,40 +10,75 @@ on:
+        role-duration-seconds: 900
+        aws-region: us-east-1
+
+    - name: Login to ECR
+      uses: aws-actions/amazon-ecr-login@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    # Restore local cache
+    - name: Restore cache
+      uses: actions/cache@v3
```

actionlint (1.7.10)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)